jenkins/scripts/artifactory/utils.sh

```diff
@@ -21,6 +21,7 @@ rt_upload_artifact() {
   ANONYMOUS="${3:-1}"
 
   _CMD="curl \
+    --fail-with-body \
     $( ([[ "${ANONYMOUS}" != 1 ]] && echo " -u${RT_USER:?}:${RT_TOKEN:?}") || true) \
```

Comment:
```suggestion
    $( ([[ "${ANONYMOUS}" -ne 1 ]] && echo " -u${RT_USER:?}:${RT_TOKEN:?}") || true) \
```
Let's fix also the comparison while at it. Numbers are compared with `-ne` and `-eq`, not `==` and `!=`.